apps/roboshield/src/components/MobileNavBar/MobileNavBar.tsx

```diff
@@ -11,7 +11,7 @@ import {
 import { styled } from "@mui/material/styles";
 import React, { ForwardedRef } from "react";
 
-import menuIcon from "@/roboshield/assets/icons/menu-icon.svg?url";
+import menuIcon from "@/roboshield/assets/icons/menu-icon.svg";
```

Comment:
```suggestion
import MenuIcon from "@/roboshield/assets/icons/Type=menu, Size=32, Color=CurrentColor.svg";
```

`Color` could be `White`, not too sure.

PS: It's weird that it's failing because we seem to have configured webpack correctly.